### DISCUSSION TITLE
【求助】从没见过的新型空洞？！

### DISCUSSION THREAD

**Discussion by M1aoW** [share121/inter-knot#2914] [其他]
![6569fmg8](https://github.com/user-attachments/assets/ba5be7d4-a815-4898-8f49-e5d3bcba0ec9)
![f0c3qhbf](https://github.com/user-attachments/assets/563e787d-2427-435e-8991-ee3fd94ab200)
在绳网上看到好多同遭遇的帖子了，目前这种空洞我没有在任何渠道上见过或者听说过。
整合信息，也许任何空洞都有可能掉入这里，最重要的是一般萝卜根本记录不到这个空洞的裂隙，请绳匠和盗洞客们务必小心！
让后就是求助各位绳匠，治安局也行，我只是一位想要感受刺激而误入空洞探险的普通人，就当我是盗洞客吧，我自首，快来救我出去！！！能救我出去的绳匠必有重谢！

**Comment by LuAn-afl** [comment]
太好了是后室我们死定了

**Comment by M1aoW** [comment]
汇报目前情况，我在这个名为后室的空洞走了近半小时，依旧重复着一开始的场景，没有见到任何生物，甚至那种在空洞内被以太侵蚀的感觉都没有，头顶白炽灯的噪音使我感到烦躁，我现在感到很累，这实在是太不走运了，我见到有些人对这个空洞内部有所了解，希望这些人能够积极对我们提供帮助，我的手机的电量不多了，希望能撑到救援的到来。

**Reply by gold-shooter** [reply]
丸辣

**Comment by Lcmagi** [comment]
永远怀念楼主

**Comment by Jobson1950** [comment]
这个委托我接了（才怪）

**Comment by lllliufeng** [comment]
丸辣

**Comment by firefly26710bite** [comment]
丸辣

**Comment by HSTTH** [comment]
丸辣
